recipes/gtk/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set major_minor = "2.24" %}
+{% set version = major_minor + ".31" %}
+
+package:
+  name: gtk
+  version: {{ version }}
+
+source:
+  url: http://ftp.gnome.org/pub/GNOME/sources/gtk+/{{ major_minor }}/gtk+-{{ version }}.tar.xz
+  sha256: 68c1922732c7efc08df4656a5366dcc3afdc8791513400dac276009b40954658
+
+build:
+  number: 0
+  skip: true  # [win]
+  # Requires Cairo's X11 libs.
+  # Please see issue ( https://github.com/conda-forge/cairo-feedstock/issues/22 ).
+  # Also please see PR ( https://github.com/conda-forge/cairo-feedstock/pull/23 ).
+  skip: true  # [osx]
```

Comment:
Getting macOS support will require Cairo having X11 libraries. Please see issue ( https://github.com/conda-forge/cairo-feedstock/issues/22 ) and PR ( https://github.com/conda-forge/cairo-feedstock/pull/23 ) for more details.